src/cobra/test/test_flux_analysis/test_deletion.py

```diff
@@ -42,7 +42,7 @@ def test_single_gene_deletion_fba(model, all_solvers):
         model=model, gene_list=list(growth_dict), method="fba", processes=1
     )["growth"]
     for gene, value in iteritems(growth_dict):
-        assert np.isclose(result[frozenset([gene])], value, atol=1e-02)
+        assert np.isclose(result[[(gene,)]], value, atol=1e-02)
```

Comment:
For all of these I don't understand why they need to be wrapped in a list.

Reply:
Will get unpacked automatically by pandas otherwise into a string and index. Deletion module indices are always tuples even when being single string indices.